Doc/library/datetime.rst

```diff
@@ -1216,13 +1216,13 @@ Instance methods:
 .. method:: datetime.isoformat(sep='T', timespec='auto')
 
    Return a string representing the date and time in ISO 8601 format,
-   YYYY-MM-DDTHH:MM:SS.mmmmmm or, if :attr:`microsecond` is 0,
+   YYYY-MM-DDTHH:MM:SS.ffffff or, if :attr:`microsecond` is 0,
    YYYY-MM-DDTHH:MM:SS
 
-   If :meth:`utcoffset` does not return ``None``, a 6-character string is
-   appended, giving the UTC offset in (signed) hours and minutes:
-   YYYY-MM-DDTHH:MM:SS.mmmmmm+HH:MM or, if :attr:`microsecond` is 0
-   YYYY-MM-DDTHH:MM:SS+HH:MM
+   If :meth:`utcoffset` does not return ``None``, a string is
+   appended, giving the UTC offset:
+   YYYY-MM-DDTHH:MM:SS.ffffff+HH:MM[:SS[.ffffff]] or, if :attr:`microsecond`
+   is 0 YYYY-MM-DDTHH:MM:SS+HH:MM[:SS[.ffffff]].
```

Comment:
First, please remove ffffff from the 0 microseconds case;  second, don't repeat the datetime part in the offset string; and finally, add a "," after 0.  (I would rather use —'s instead of commas, but this is a matter of taste and switching would require a full review of similar language in datetime docs.)

Reply:
I don't understand your comment. The 0 microsecond case is relative to the microsecond attribute of the datetime, not the offset. That's why the datetime part is repeated as it varies based on the value of its microsecond attribute.

Reply:
I see.  Can this be rewritten so that it just focuses on the TZ part?

Reply:
On the second thought, this is fine as is.